### DISCUSSION TITLE
How can I delete bind for `<tab>`

### DISCUSSION THREAD

**Discussion by mrxiaozhuox** [LazyVim/LazyVim#4971] [Q&A] [answered]
<img width="979" alt="截屏2024-12-09 00 16 28" src="https://github.com/user-attachments/assets/bd415ae9-150d-4dde-b670-0a8df5c924ac">

There have a `<tab>` for toggle terminal, but I want cancel it,  but keymap.del does not work:
```
vim.keymap.del("n", "<tab>")
vim.keymap.del("t", "<tab>")
```

**Comment by folke** [comment]
Tab is not a key binding set by LazyVim, so you have that somewhere in your config

**Reply by mrxiaozhuox** [reply]
> Tab is not a key binding set by LazyVim, so you have that somewhere in your config

I didn't config any other mapping.
<img width="1154" alt="截屏2024-12-09 01 31 13" src="https://github.com/user-attachments/assets/ef12c058-e433-486d-a85d-9282f3fbc326">
<img width="306" alt="截屏2024-12-09 01 31 29" src="https://github.com/user-attachments/assets/59deed24-fd00-4ae4-bee2-a7cf9836420b">
I just use default config and covered some plugin settings.

**Reply by dpetka2001** [reply]
Try `verbose nmap <tab>` to show you from where it's set. It's either some setting in your personal configuration or from a plugin you installed. That's not a default LazyVim mapping. You can try installing a default LazyVim installation from scratch without any of your personal settings and you can see for yourself.

**Reply by mrxiaozhuox** [reply]
> Try `verbose nmap <tab>` to show you from where it's set. It's either some setting in your personal configuration or from a plugin you installed. That's not a default LazyVim mapping. You can try installing a default LazyVim installation from scratch without any of your personal settings and you can see for yourself.

OK, I `verbose nmap <tab>` doesn't have anything, I will check my plugin.

**Reply by mrxiaozhuox** [reply, marked as answer]
OK, I found the problem: 
https://stackoverflow.com/questions/23258920/what-is-the-reason-that-ctrli-and-tab-keys-send-same-codes
<c-i> will have a same bind with <tab>, this surprised me very much.
